photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriver.scala

```diff
@@ -473,7 +475,10 @@ object GameTrainingDriver extends GameDriver {
       featureShards: Iterable[FeatureShardId]): FeatureShardStatistics =
     featureShards.map { featureShardId =>
       // Calling rdd explicitly here to avoid a typed encoder lookup in Spark 2.1
-      (featureShardId, BasicStatisticalSummary(data.select(featureShardId).rdd.map(_.getAs[SparkVector](0))))
+      (featureShardId, BasicStatisticalSummary(
+        data.select(featureShardId)
+          .rdd
+          .map(row => Vectors.fromML(row.getAs[SparkMLVector](0)))))
```

Comment:
Similar to above: should add a TODO for when `ml.Vector` is the Photon ML standard.